docs/core/compatibility/core-libraries/6.0/snippets/ReadStream.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.IO.Compression;
```

Comment:
this should be in a subfolder.

Reply:
Why can't all the snippets for the .NET 6 breaking changes for core .NET libraries be in one project?